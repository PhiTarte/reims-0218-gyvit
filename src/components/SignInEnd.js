//TODO CREATE SIGN IN TITLE
import React from "react"

const SignInEnd = () => (
  <div>
    <p>Don't have an account? Get started </p>
  </div>
)

export default SignInEnd
