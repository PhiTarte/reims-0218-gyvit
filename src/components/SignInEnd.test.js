//TODO CREATE TEST

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import SignInEnd from "./SignInEnd"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<SignInEnd />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<SignInEnd />).toJSON()
  expect(tree).toMatchSnapshot()
})
