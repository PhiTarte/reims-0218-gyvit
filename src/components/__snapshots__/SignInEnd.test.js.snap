// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div>
  <p>
    Don't have an account? Get started 
  </p>
</div>
`;
